src/python/espressomd/inertia_tensor.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+
+
+def inertia_tensor(positions, masses):
+    """
+    Calculate the moment of inertia tensor for given point masses
```

Comment:
Isn't that thing called just `inertia tensor` ? `moment of inertia tensor` sounds odd, because the moment of inertia is a vector. C.f. `§32 The inertia tensor` from [Scripture](http://renaissance.ucsd.edu/courses/mae207/mech.pdf).

Reply:
I adopted the nomenclature from [wikipedia](https://en.wikipedia.org/wiki/Moment_of_inertia#Inertia_tensor) but maybe you are right

Reply:
I think `intertia tensor` is just short for `moment of intertia tensor`, I think what you did is fine, it was late...